src/custom/utils/misc.ts

```diff
@@ -91,3 +91,20 @@ export function getTokensFromMarket<T>({
     }
   }
 }
+
+/**
+ * Basic hashing function
+ */
+export function hashCode(text: string): number {
```

Comment:
There's a hashing lib added recently by Ramiro. You could use that instead of writing your own

Reply:
We don't need anything to fancy, I used https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript but I will review

Reply:
Can u point me to the hashing lib he used. Is it `multihash`? 

Im not too concern using this simple one either. is pretty fast. Happy to review this, although giving we want this release tomorrow, probably I wouldn't overcomplicate too much.

Reply:
https://github.com/gnosis/cowswap/blob/866181b4af7fd705d56e7597d2191692075d1204/src/custom/utils/deterministicHash.ts#L9-L8

The hashing lib used comes from `ethersjs`